pkgs/development/libraries/qt-5/modules/qtbase.nix

```diff
@@ -146,6 +146,11 @@ stdenv.mkDerivation {
             sed -i mkspecs/common/linux.conf \
                 -e "/^QMAKE_INCDIR_OPENGL/ s|$|${libGL.dev or libGL}/include|" \
                 -e "/^QMAKE_LIBDIR_OPENGL/ s|$|${libGL.out}/lib|"
+          '' +
+        lib.optionalString (!stdenv.hostPlatform.isx86_64)
```

Comment:
Probably should use `isx86_32` and also check that `stdenv.cc.isGnu`

Reply:
s/isx86_64/isi686/

Reply:
`stdenv.hostPlatform.isx86_32` looks good to me.

Reply:
:+1: